includes/class-wc-payments-account.php

```diff
@@ -2364,7 +2364,7 @@ function () {
 	 * @return void
 	 */
 	public function update_account_data( $property, $data ) {
-		$account_data = $this->database_cache->get( Database_Cache::ACCOUNT_KEY );
+		$account_data = $this->database_cache->get( Database_Cache::ACCOUNT_KEY, true );
```

Comment:
Since we want to directly write a modified version of the cache account data, it makes sense not to try to refresh it.

Reply:
That totally makes sense 👍